my.css

```diff
@@ -352,3 +352,650 @@ body {
+ justify-content: space-between;
+ width: 140px;
+ height: 29px;
+  }
+  .card__choice {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-top: 15px;
+  }
+  .card__choice-buttons {
+  display: none;
+   }
+   .card__row {
+    width: 275px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 5px;
+  }
+   .card__button {
+  width: 25px;
+  height: 25px;
+  border-radius: 100%;
+  border: unset;
+  
+  }
+ .card__basket-button {
+  width: 100%;
+  height: 37px;
+  border: 1px solid #000;
+  box-sizing: border-box;
+  background-color: #fff;
+  }
+}
+
+/* Третья секция */
+
+.sectionlock {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  min-height: 900px;
+  background: linear-gradient(90deg, rgba(235,227,216,1) 11%,
+              rgba(235,227,216,1) 16%, rgba(235,227,216,1) 22%, 
+              rgba(0,0,0,1) 22%, rgba(0,0,0,1) 26%, 
+              rgba(235,227,216,1) 26%);
+}
+.sectionlock__content {
+  display: flex;
+  max-width: 1150px;
+  width: 100%;
+  margin: 0 auto;
+}
+.sectionlock__img {
+  width: 60%;
+  height: 844px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+.sectionlock__img-clothes {
+  width: 780px;
+  height: 600px;
+  box-shadow:0px 5px 18px 0px rgba(0,0,0,0.43);
+}
+.sectionlock__iner {
+ width: 42%;
+ display: flex;
+ flex-direction: column;
+ justify-content: center;
+}
+.total {
+  width: 475px;
+  padding: 46px 20px 45px 55px; 
+  margin: 0 auto;
+  display: flex;
+  flex-flow: column;
+  background-color: #fae8e4;
+  border: 1px solid #fae8e4;
+  box-sizing: border-box;
+}
+.total__text {
+  padding-top: 5px;
+  padding-bottom: 30px;
+  margin: 0 auto;
+  opacity: 0.5;
+}
+.total__choice {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 40px;
+  padding-top: 14px;
+}
+
+.total__choice-buttons {
+  border: 1.5px solid #000;
+  width: 95px;
+  height: 40px;
+  display: flex;
+  justify-content: space-around;
+}
+.total__choice-buton {
+  box-sizing: border-box;
+  background-color: #fae8e4;
+  border: #fff;
+}
+.total__choice-figure {
+  display: flex;
+  align-items: center;
+  
+}
+.total__basket{
+ display: flex;
+ width: 283px;
+ height: 44px;
+ align-items: center;
+ justify-content: center;
+}
+.total__basket-buttons {
+  width: 100%;
+  height: 43px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+  background-color: #fae8e4;
+}
+.sectionlock__footer {
+  display: flex;
+  justify-content: flex-end;
+  padding-top: 30px;
+}
+.sectionlock__footer-img {
+  padding-left: 15px;
+}
+.footer {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+@media screen and (max-width: 1130px) {
+     .sectionlock {
+    max-height: 900px;
+    max-width: 1150px;
+    position: relative;
+     }
+     .sectionlock__img-clothes {
+  width: 700px;
+  height: 640px;
+    }
+     .sectionlock__iner {
+     width: 35%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)